editor/editor_node.cpp

```diff
@@ -5808,6 +5831,9 @@ void EditorNode::_bind_methods() {
 	ClassDB::bind_method("_close_messages", &EditorNode::_close_messages);
 	ClassDB::bind_method("_show_messages", &EditorNode::_show_messages);
 
+	ClassDB::bind_method("store_object_properties", &EditorNode::store_object_properties);
```

Comment:
IMO This function should only be called if you have a script, so the intention is better. Otherwise you have to go to the function and see that it does nothing if it has a script.

Reply:
Nah, the function is supposed to store available script properties. If there is no script, it just has no properties to store.

It makes the code much simpler. Otherwise I'd have to wrap each call and make it less readable.